### ISSUE TITLE
Suggestion: config map

### ISSUE THREAD

**Issue by anandkumarpatel** [thejsj/auto-kubernetes-lets-encrypt#2]
I think it might be better to change all optional values to is a config map. That way people can use the same deployment and only have to change the config file (config map for params, secret for the secret key)

**Comment by thejsj** [comment]
Yeah, the nice part about doing that is that you can get rid of the `.bash` file.
